appdev/guides/pushnotifications.rst

```diff
@@ -0,0 +1,235 @@
+Push Notifications
+==================
+
+Let's assume that you have an app created with Clickable and published on the OpenStore. But now you want to be able to send Push Notifications to your users. First of all, you need to understand how this is working:
+
+.. image:: pushoverview.jpg
```

Comment:
You have excellent handwriting and this chart makes sense visually. However, I would prefer that charts in the documentation are made using a computer graphics tool such as https://draw.io.

Reply:
You are right of course. :D I will create one

Reply:
Its done now =)